adazza-swagger.gradle

```diff
@@ -0,0 +1,32 @@
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "gradle.plugin.org.detoeuf:swagger-codegen-plugin:1.7.4"
+    }
+}
+
+apply plugin: org.detoeuf.SwaggerCodeGenPlugin
+
+sourceSets {
+    main {
+        java {
+            srcDir file("${project.buildDir.path}/swagger/src/main/java")
```

Comment:
for this to look like scrooge it would be:
`buildDir/generated-sources/` the src/main/java is superfluous.